### DISCUSSION TITLE
ilogtail多行日志采集只适用于k8s的标准输出吗？容器内日志多行日志不支持？

### DISCUSSION THREAD

**Discussion by onejimmyboy** [alibaba/ilogtail#1348] [Help] [answered]
version:1.8.4
测了一下官网上给的日志信息，然后写进了容器日志问题，但是ilogtail采集的时候并没的当成多行采集，仍然是单行，下面看的是ilogtail的日志。
![image](https://github.com/alibaba/ilogtail/assets/57432151/bab07c5d-563a-4dc4-a949-4bba484ac256)
备注：下面截图中BeginLineRegex: "\\{"，这个是第二次测试，第一次测试的时候完全是上图的日志和 BeginLineCheckLength: 10     BeginLineRegex: "\\d+-\\d+-\\d+.*"。
![image](https://github.com/alibaba/ilogtail/assets/57432151/be765193-bbc6-4dba-9b68-336a659e4f79)
![image](https://github.com/alibaba/ilogtail/assets/57432151/a0c19d7d-f1b9-45be-ae22-8212fcae1d91)

**Comment by messixukejia** [comment, marked as answer]
参数要查看对应插件的文档，不能混着用
https://ilogtail.gitbook.io/ilogtail-docs/plugins/input/input-file#cai-ji-jian-dan-duo-hang-ri-zhi

后面stdout会提供c++版的插件，到时候参数会进行统一

**Reply by Shylcok** [reply]
这个是2.0.0的文档吗 1.8.4没有解决方案吗

**Reply by Shylcok** [reply]
processor_regex_accelerate 这个插件

**Reply by 7y-9** [reply]
https://ilogtail.gitbook.io/ilogtail-docs/v/1.8.0/data-pipeline/processor/processor-split-log-regex
可以试下这个 processor_split_log_regex

**Reply by Shylcok** [reply]
多谢，我试试看
